src/utils/hashes/sha512Half.ts

```diff
@@ -1,11 +1,19 @@
 import { createHash } from "crypto";
 
-const sha512Half = (hex: string): string => {
+const HASH_SIZE = 64;
+
+/**
+ * Compute a sha512Half Hash of a hex string.
+ *
+ * @param hex - Hex string to hash.
+ * @returns Hash of hex.
+ */
+function sha512Half(hex: string): string {
```

Comment:
This exists in ripple-binary-codec as well, right? Would it be better to just import this method from there instead of duplicating it here?

Reply:
Yeah, it is, but its not exported at the top level of the `codec`, so IMO it should stay in ripple-lib.